ts/src/bitteam.ts

```diff
@@ -0,0 +1,2395 @@
+        }, market);
+    }
+
+    async fetchTrades (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Trade[]> {
+        /**
+         * @method
+         * @name bitteam#fetchTrades
+         * @description get the list of most recent trades for a particular symbol
+         * @see https://bit.team/trade/api/documentation#/CMC/getTradeApiCmcTradesPair
+         * @param {string} symbol unified symbol of the market to fetch trades for
+         * @param {int} [since] timestamp in ms of the earliest trade to fetch
+         * @param {int} [limit] the maximum amount of trades to fetch
+         * @param {object} [params] extra parameters specific to the bitteam api endpoint
+         * @returns {Trade[]} a list of [trade structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#public-trades}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'pair': market['id'],
+        };
+        const response = await this.publicGetTradeApiCmcTradesPair (this.extend (request, params));
+        //
+        //     [
+        //         {
+        //             "trade_id": 34970337,
+        //             "price": 37769.994793,
+        //             "base_volume": 0.00119062,
+        //             "quote_volume": 44.96971120044166,
+        //             "timestamp": 1700827234000,
+        //             "type": "buy"
+        //         },
+        //         {
+        //             "trade_id": 34970347,
+        //             "price": 37769.634497,
+        //             "base_volume": 0.00104009,
+        //             "quote_volume": 39.28381914398473,
+        //             "timestamp": 1700827248000,
+        //             "type": "buy"
+        //         },
+        //         ...
+        //     ]
+        //
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    async fetchMyTrades (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitteam#fetchMyTrades
+         * @description fetch all trades made by the user
+         * @see https://bit.team/trade/api/documentation#/PRIVATE/getTradeApiCcxtTradesofuser
+         * @param {string} symbol unified market symbol
+         * @param {int} [since] the earliest time in ms to fetch trades for
+         * @param {int} [limit] the maximum number of trades structures to retrieve (default 10)
+         * @param {object} [params] extra parameters specific to the bitteam api endpoint
+         * @returns {Trade[]} a list of [trade structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#trade-structure}
+         */
+        await this.loadMarkets ();
+        const request = {};
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['pairId'] = market['numericId'];
```

Comment:
aren't we missing .toString() here? does that need we don't need to stringify the id? 😅 if the exchange accepts it as an integer we can remove all .toString()

Reply:
This current endpoint accepts `pairId` as number. Others - as a string )